packages/mdc-checkbox/_mixins.scss

```diff
@@ -69,9 +69,7 @@
 
 @mixin mdc-checkbox-ink-color($color) {
   .mdc-checkbox__checkmark__path {
-    // !important is needed here because a stroke must be set as an attribute on the SVG in order
-    // for line animation to work properly.
-    @include mdc-theme-prop(stroke, $color, $important: true);
+    @include mdc-theme-prop(stroke, $color);
```

Comment:
does line animation still work?

Reply:
Nope - removing this `!important` breaks the line animation when toggling from `indeterminate` to `determinate` on an unchecked checkbox ☹️ 

If you slow animations down to 10% in the Chrome Dev Tools, and click "Toggle indeterminate" near an unchecked checkbox, you can see it:

![pr 1987 indeterminate bug](https://user-images.githubusercontent.com/409245/35258400-f9338dd0-ffb3-11e7-9f22-8486507896c7.gif)

It's supposed to look like this:

![pr 1987 indeterminate prod](https://user-images.githubusercontent.com/409245/35258437-59b11088-ffb4-11e7-8cd9-9dd64d4b6ed3.gif)

Reply:
Good catch. I'll find a different way to accomplish this.

Reply:
FYI I think your comments apply to the opposite lines. Just for future reference. Gonna link them.

Reply:
https://github.com/material-components/material-components-web/pull/1987#discussion_r163142706